core/src/main/resources/error/error-classes.json

```diff
@@ -490,7 +490,7 @@
   },
   "UNRESOLVED_MAP_KEY" : {
     "message" : [
-      "Cannot resolve column <columnName> as a map key. If the key is a string literal, please add single quotes around it."
+      "Cannot resolve column <objectName> as a map key. If the key is a string literal, please add single quotes around it."
```

Comment:
I renamed it because there is another error class which shares the same function in source code, and it has the same structure and the first parameter `objectName`. Having the same name simplifies the implementation.

Reply:
Note: we will not get away with renaming parameter names in the long run, once third parties start building dependencies on the API.